src/parameter_indexing.jl

```diff
@@ -15,6 +15,56 @@ parameter_values(arr::AbstractArray) = arr
 parameter_values(arr::AbstractArray, i) = arr[i]
 parameter_values(prob, i) = parameter_values(parameter_values(prob), i)
 
+"""
+    parameter_values_at_time(p, t)
+
+Return an indexable collection containing the value of all parameters in `p` at time index
+`t`. This is useful when parameter values change during the simulation
+(such as through callbacks) and their values are saved. `t` is the time index in the
```

Comment:
> `t` is the time index

This sounds ambiguous, is `t` time or index?

If it's an index, perhaps we could clarify by calling it `ti` and say that it's an integer?

Reply:
I'll refactor it to be `i`

Reply:
Fixed in https://github.com/SciML/SymbolicIndexingInterface.jl/pull/53/commits/cc5da9083caa15e894a4ea2f8d35300b0fb27ff4